config/base.yaml

```diff
@@ -112,6 +112,48 @@ bert:
     --model: "Bert"
     --batch-size: 32
 
+fp32:
```

Comment:
* I think I'd prefer naming the benchmark `bert-fp32` (etc.)
* We should remove the plain `bert` benchmark, because it's redundant with the ones we are adding here.
* Specify a batch size for all of them.

Reply:
done